src/drafter.h

```diff
@@ -62,43 +63,73 @@ typedef refract::IElement drafter_result;
 #endif
 
 
+/* Serialization formats, currently only YAML or JSON */
 typedef enum {
     DRAFTER_SERIALIZE_YAML = 0,
     DRAFTER_SERIALIZE_JSON
 } drafter_format;
 
+/* Parsing options
+ * - requireBlueprintName : API has to have a name, if not it is a parsing error
+ */
 typedef struct {
     bool requireBlueprintName;
 } drafter_parse_options;
 
-// TODO: Maybe rename this to drafter_serialize_options or combine both
+
+/* Serialization options
+ * - sourcemap : Include sourcemap in the serialized result
+ * - format : Serialization format see above
+ */
 typedef struct {
     bool sourcemap;
     drafter_format format;
-} drafter_options;
+} drafter_serialize_options;
+
 
-/* Parse API Blueprint and serialize it to given format.*/
-// TODO: Add parse options to this
+/* Error codes */
+#define DRAFTER_EUNKNOWN -1
```

Comment:
What about putting these in an enumeration?

```c
enum drafter_result_t: int {
    DRAFTER_OK -1
    DRAFTER_EUNKNOWN = -1
    DRAFTER_EINVALID_INPUT = -2
    DRAFTER_EINVALID_OUTPUT = -3
}
```

That away they are grouped together and allow proper type safety in the result.